network/hostport/hostport_manager.go

```diff
@@ -31,7 +31,6 @@ import (
 	"github.com/sirupsen/logrus"
 	v1 "k8s.io/api/core/v1"
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
-	iptablesproxy "k8s.io/kubernetes/pkg/proxy/iptables"
```

Comment:
Shouldn't this be replaced by "k8s.io/kubernetes/pkg/proxy/iptables" instead? The constant seems defined there - https://github.com/kubernetes/kubernetes/blob/bae2c62678db2b5053817bc97181fcc2e8388103/pkg/proxy/iptables/proxier.go#L71